src/db/db_user.erl

```diff
@@ -0,0 +1,32 @@
+-module(db_user).
+
+-export([create/1,
+         authenticate/2,
+         get_from_countries/1]).
+
+create(#{<<"email">> := Email,
```

Comment:
what about the stuff coming out of the DB? I guess we should be converting keys to atoms there too